### ISSUE TITLE
Transfer this repository to nix-community?

### ISSUE THREAD

**Issue by benley** [nix-community/nix-emacs#53]
Following up on https://github.com/travisbhartwell/nix-emacs/pull/45#issuecomment-803785795:

I've been added to the [nix-community](https://github.com/nix-community) github org, so I think if @travisbhartwell initiates an ownership transfer of this repository to nix-community I'll be able to (a) accept the transfer, and (b) re-add @travisbhartwell as a repository maintainer (if necessary - this part may happen automatically).

Related github docs: https://docs.github.com/en/github/administering-a-repository/transferring-a-repository#transferring-a-repository-owned-by-your-user-account

**Comment by jcs090218** [comment]
Hi, I was searching company backend for nix expressions then I found this amazing package!

Does this package need a maintainer (by seeing comment https://github.com/travisbhartwell/nix-emacs/pull/45#issuecomment-803785795)? If yes, I would love to help out and co-maintain with others!

A little bit of myself, I have created 40+ elisp packages and co-maintain 70+ of them. I would say I am much familiar with Emacs Lisp! But I am new to nix language and still in the process of learning it! I think it would be a great chance for me to start with something I'm familiar with! Therefore, if someone is more familiar with nix can help, that would be awesome! :)

cc @benley @travisbhartwell
